include/Stand_der_Technik.tex

```diff
@@ -64,5 +65,6 @@ \section{Technische Konzepte}
 \end{table}
 
 \section{Stand im Bibliothekswesen}
-%TODO Pascal
+Im Bibliothekswesen ist RFID immer häufiger anzutreffen. Der Hauptvorteil liegt hier darin, dass Personalkosten eingespart werden können. Diese Einsparungen wird erreicht, durch mehrere nun automatisierte Prozesse, wie die selbst ausleihe eines Kunden.
+Diese Komplett-Systeme werden von verschiedenen Anbietern geliefert. In der Schweiz liefern diese meist RFID HF Technologie aus. Wie in Kapitel \ref{sec:technischeKonzepte} beschrieben beträgt die maximale erreichbare Distanz von RFID HF knapp einen Meter. Diesem Umstand ist wahrscheinlich zu verdanken, dass keine der gefunden Komplett-Systemanbieter, Identifikation von HF Tags in einem Hochregallager anbietet \parencite{Keller2010}.  
```

Comment:
Dies bezieht sich jetzt ja nur auf eine Quelle, und nur aus der Schweiz. Wie sieht es mit anderen Bibliotheken und Technologien aus? (Zum Beispiel die City University of Hong Kong)

Reply:
Wie oben erwähnt gehe ich bewusst nur auf HF Tags ein, da es bei UHF sehr sehr viele Hersteller gibt, welche dies ermöglichen (nicht nur die University of Hong Kong). Es also mit UHF sehr wohl funktionieren könnte, auch wenn bei UHF die Interferenz zwischen Tags stärker ist (kommt auf den Lesewinkel an). Bei diesen Komplettanbietern sieht es jedoch so aus, dass diese etwas mehr Zeit für das Lesen der Tags erhalten (was uns wahrscheinlich auch helfen könnte).